python/paddle/fluid/layers/nn.py

```diff
@@ -8425,3 +8426,52 @@ def bilinear_tensor_product(x,
 
     # add activation
     return helper.append_activation(out)
+
+
+@templatedoc()
+def tree_conv(nodes_vector,
+              edge_set,
+              output_size,
+              name=None,
+              num_filters=1,
+              max_depth=2,
+              act='tanh',
+              param_attr=None,
+              bias_attr=None):
+    """
+    ${comment}
+
+    Args:
+        nodes_vector(${nodes_vector_type}): ${nodes_vector_comment}
+        edge_set(${edge_set_type}): ${edge_set_comment}
+        output_size(int): output feature width
+        num_filters(int, default 1): number of filters
```

Comment:
For doc style consistency， we generally make it like "num_filters(int): number of filters, default 1"